core/src/main/java/org/apache/iceberg/actions/RewriteFileGroup.java

```diff
@@ -48,14 +51,14 @@ public RewriteFileGroup(
       List<FileScanTask> fileScanTasks,
       int outputSpecId,
       long writeMaxFileSize,
-      long splitSize,
+      long inputSplitSize,
```

Comment:
Why rename here?

Reply:
I have forgot to rename these in my previous PR. During the review we changed from `splitSize` to `inputSplitSize` to highlight that this is for the input. We did the rename in the parent class (`RewriteGroupBase`), but I didn't do it here. Since this is a new class introduced by my previous PR, I felt that we should keep the naming consistent.